luda-editor/client/src/pages/sequence_edit_page/components/wysiwyg_editor/mod.rs

```diff
@@ -71,36 +95,96 @@ impl Component for WysiwygEditor<'_> {
+                                    update_sequence_with_mover(context, graphic_index);
+                                }
                             }
                         }
                         set_dragging.set(None);
                     }
+                    Event::KeyDown { event } => {
+                        let arrow_key_down = matches!(
+                            event.code,
+                            Code::ArrowUp | Code::ArrowLeft | Code::ArrowRight | Code::ArrowDown
+                        );
+                        if let Some(graphic_index) = *editing_graphic_index {
+                            if cut_editor_focused && arrow_key_down {
+                                let set_new_dragging = || {
+                                    let moving_with = match event.code {
+                                        Code::ArrowLeft => MovingWith::KeyLeft,
+                                        Code::ArrowRight => MovingWith::KeyRight,
+                                        Code::ArrowUp => MovingWith::KeyUp,
+                                        Code::ArrowDown => MovingWith::KeyDown,
+                                        _ => unreachable!(),
+                                    };
+                                    set_dragging.set(Some(Dragging::Mover {
+                                        context: MoverDraggingContext {
+                                            start_global_xy: Xy::zero(),
+                                            end_global_xy: Xy::zero(),
+                                            container_wh: wh,
+                                            moving_with,
+                                        },
+                                    }));
+                                };
+                                let mutate_dragging = || {
+                                    let mut delta_xy = match event.code {
+                                        Code::ArrowLeft => Xy::new(-(1.0.px()), 0.0.px()),
+                                        Code::ArrowRight => Xy::new(1.0.px(), 0.0.px()),
+                                        Code::ArrowUp => Xy::new(0.0.px(), -(1.0.px())),
+                                        Code::ArrowDown => Xy::new(0.0.px(), 1.0.px()),
+                                        _ => unreachable!(),
+                                    };
+                                    if !namui::keyboard::shift_press() {
+                                        delta_xy *= 10.0;
+                                    }
+                                    set_dragging.mutate(move |dragging| {
+                                        if let Some(Dragging::Mover { context }) = dragging {
+                                            context.end_global_xy =
+                                                context.end_global_xy + delta_xy;
+                                        }
+                                    });
+                                };
```

Comment:
does this speed depend on frame rate?

Reply:
Operates whenever a key event occurs.
I think it depends on how often the web browser repeats key events.
And I think it's probably related to operating system's key repetition setting.